Modified to existing check to support "UiPath Coding Standards and Wo…

aubridge-miyaichi opened the pull request:
…rkflow Quality Evaluation Kit"

1.HardcodedPassword.xaml
Changed to retrieve attribute list to check from argument so that attributes other than "Password" can be checked.

2.ProjectJsonConfigurationFile.xaml
Added the check that the workflow file of "main" parameter of project.json exists.

mshcruz commented:
Thinking a bit more, it's probably better to create a different check for the Main.xaml file as well, because it now does two things and the user cannot know which one made it fail (i.e., if the project.json doesn't exist or if the main workflow doesn't exist).
If it is kept like this, it's probably necessary to change the name of the check a bit to include the second part of the check.

aubridge-miyaichi commented:
- HardcodedPassword.xaml:

1. The file cannot be opened correctly in Studio 2018.2.6...
  Delete CurrentIndex attribute.
2. The purpose of the modifications to this workflow...
  Discard the fix and created "Hardcoded Property".

- ProjectJsonConfigurationFile.xaml:
1. The last sentence in the annotation...
  Fixed.
2. This file contains a typo...
  Fixed.
3. It's not clear why the TryCatch activity is being used...
  Delete the TryCatch activity and pass the JSON deserialization exception to the caller.
4. The display names of TryCatch...
  Delete the TryCatch activity.
5. It's unnecessary to use Assign False..
  Use False as its default value.
6. Change the condition of the last If activity...
  Clarified conditions.
7. In English Checklist.xlsx...
  Fixed English and Japanese versions of Checklist.xlsx.
8. Thinking a bit more, it's probably better to create a different check...
  Splitting the check is under consideration.

mshcruz commented:
I added these changes to #18, so I'll close this PR.
